hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOmSnapshot.java

```diff
@@ -194,6 +204,69 @@ private void init() throws Exception {
 
     // stop the deletion services so that keys can still be read
     keyManager.stop();
+    preFinalizationChecks();
+    finalizeOMUpgrade();
+  }
+
+  private static void expectFailurePreFinalization(LambdaTestUtils.
+      VoidCallable eval) throws Exception {
+    OMException ex  = Assert.assertThrows(OMException.class,
+            () -> eval.call());
+    Assert.assertTrue(ex.getMessage().contains(
+            "cannot be invoked before finalization."));
+  }
```

Comment:
```suggestion
  private static void expectFailurePreFinalization(LambdaTestUtils.
      VoidCallable eval) {
    OMException ex = Assert.assertThrows(OMException.class, eval::call);
    Assert.assertEquals(NOT_SUPPORTED_OPERATION_PRIOR_FINALIZATION,
        ex.getResult());
    Assert.assertTrue(ex.getMessage().contains(
        "cannot be invoked before finalization."));
  }
```

Reply:
yup, updated to check error code of "NOT_SUPPORTED_OPERATION_PRIOR_FINALIZATION". And indentation updated.